lib/spack/spack/cmd/install.py

```diff
@@ -48,6 +48,9 @@ def setup_parser(subparser):
         '-n', '--no-checksum', action='store_true', dest='no_checksum',
         help="Do not check packages against checksum")
     subparser.add_argument(
+        '-p', '--parallel', action='store_true', dest='build_parallel',
```

Comment:
@mathstuf I think `-j` is already passed to `make` or equivalent tools. The added option `-p` will allow for multiple `make -j <number-of-jobs>` from different packages to occurr at the same time. Or at least this is what I got from the description.